crates/nargo_cli/tests/test_data/8_integration/src/main.nr

```diff
@@ -242,7 +244,13 @@ fn sig1(x: u32) -> u32 {
 }
 
 
-fn main(a: [u32; 100], b: [u32; 100], c: [u32; 4], mut d: [u32; 4], m: [u8; 32]) {  
+fn main(a: [u32; 100], b: [u32; 100], c: [u32; 4], mut d: [u32; 4], m: [u32; 32]) {  
+
+    // let sha_m = (m[0] << 24 as u32)
+    //         | (m[1] << 16 as u32);
+
+    // constrain sha_m != 10;
+    
```

Comment:
@vezenovm what's happening here?

Reply:
This is residual debugging from when we were iterating on bberg with UP. Will submit a PR to clean this stuff up

Reply:
I reverted the `8_integration` test